index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Koulen&family=Luckiest+Guy&family=Poppins:wght@700&family=Special+Elite&family=Teko:wght@300&family=Yomogi&display=swap" rel="stylesheet">
+    <title>Josué</title>
+</head>
+<body>
+    <header class="Header">My Logo</header>
+    <nav class="navigation">
+    <button class="menu">
+    <i class="fa-solid fa-bars"></i></button>
+    </nav>
+    <section class="section">
+        <h1>I´m Josué Glad to see you!</h1>
+        
+    <div class="description">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
Kindly, use the appropriate `font-weight` currently it's bold. You can refer to the Figma design and match it.